avm/ptn/network/private-link-private-dns-zones/bicepconfig.json

```diff
@@ -0,0 +1,19 @@
+// This is a Bicep configuration file. It can be used to control how Bicep operates and to customize validation settings for the Bicep linter. The linter uses these settings when evaluating your Bicep files for best practices.
```

Comment:
Do we really want to start adding individual `bicepconfig.json` files that in turn mean that different modules may opt in for experimental features etc.?

Reply:
this one had to be done for the linter rule of `no-hardcoded-env-urls` unless you want me to try and supress with the next line # approach?

Reply:
Id rather have the # approach personally (also seems like less of an overhead), but happy to align if you feel strong about it 😉